ProffieOS.ino

```diff
@@ -1,1666 +1,1670 @@
-  SAVE_MODER(A, 0);
-  SAVE_MODER(B, 1);
-  SAVE_MODER(C, 2);
-  SAVE_MODER(H, 3);
-
-  // TODO enable/disable as needed
-  pinMode(boosterPin, OUTPUT);
-  digitalWrite(boosterPin, HIGH);
-#endif
-
-  Serial.begin(9600);
-#if VERSION_MAJOR >= 4
-  // TODO: Figure out if we need this.
-  // Serial.blockOnOverrun(false);
-#endif
-
-  // Wait for all voltages to settle.
-  // Accumulate some entrypy while we wait.
-  uint32_t now = millis();
-#ifdef DOSFS_CONFIG_STARTUP_DELAY
-#define PROFFIEOS_STARTUP_DELAY DOSFS_CONFIG_STARTUP_DELAY
-#else
-#define PROFFIEOS_STARTUP_DELAY 1000
-#endif
-  while (millis() - now < PROFFIEOS_STARTUP_DELAY) {
-    srand((rand() * 917823) ^ LSAnalogRead(batteryLevelPin));
-#ifdef BLADE_DETECT_PIN
-    // Figure out if blade is connected or not.
-    // Note that if PROFFIEOS_STARTUP_DELAY is smaller than
-    // the settle time for BladeDetect, this won't work properly.
-    BladeDetect.Warmup();
-#endif
-  }
-
-#ifdef ENABLE_SERIALFLASH
-  SerialFlashChip::begin(serialFlashSelectPin);
-#endif
-#ifdef ENABLE_SD
-  bool sd_card_found = LSFS::Begin();
-  if (!sd_card_found) {
-    if (sdCardSelectPin >= 0 && sdCardSelectPin < 255) {
-      STDOUT.println("No sdcard found.");
-      pinMode(sdCardSelectPin, OUTPUT);
-      digitalWrite(sdCardSelectPin, 0);
-      delayMicroseconds(2);
-      pinMode(sdCardSelectPin, INPUT);
-      delayMicroseconds(2);
-      if (digitalRead(sdCardSelectPin) != HIGH) {
-        STDOUT.println("SD select not pulled high!");
-      }
-    }
-#if VERSION_MAJOR >= 4
-    stm32l4_gpio_pin_configure(GPIO_PIN_PA5,   (GPIO_PUPD_PULLUP | GPIO_OSPEED_HIGH | GPIO_MODE_INPUT));
-    delayMicroseconds(10);
-    if (!stm32l4_gpio_pin_read(GPIO_PIN_PA5)) {
-      STDOUT.println("SCK won't go high!");
-    }
-    stm32l4_gpio_pin_configure(GPIO_PIN_PA5,   (GPIO_PUPD_PULLDOWN | GPIO_OSPEED_HIGH | GPIO_MODE_INPUT));
-    delayMicroseconds(10);
-    if (stm32l4_gpio_pin_read(GPIO_PIN_PA5)) {
-      STDOUT.println("SCK won't go low!");
-    }
-#endif
-  } else {
-    STDOUT.println("Sdcard found..");
-  }
-#endif
-
-  Looper::DoSetup();
-  // Time to identify the blade.
-  prop.FindBlade();
-  SaberBase::DoBoot();
-#if defined(ENABLE_SD) && defined(ENABLE_AUDIO)
-  if (!sd_card_found) {
-    talkie.Say(talkie_sd_card_15, 15);
-    talkie.Say(talkie_not_found_15, 15);
-  }
-#endif // ENABLE_AUDIO && ENABLE_SD
-}
-
-#ifdef MTP_RX_ENDPOINT
-
-void mtp_yield() { Looper::DoLoop(); }
-void mtp_lock_storage(bool lock) {
-  AudioStreamWork::LockSD(lock);
-}
-
-#include "mtp/mtpd.h"
-MTPD mtpd;
-
-#ifdef ENABLE_SD
-#include "mtp/mtp_storage_sd.h"
-MTPStorage_SD sd_storage(&mtpd);
-#endif
-
-#ifdef ENABLE_SERIALFLASH
-#include "mtp/mtp_storage_serialflash.h"
-MTPStorage_SerialFlash serialflash_storage(&mtpd);
-#endif
-
-#endif  // MTP_RX_ENDPOINT
-
-#include "common/clock_control.h"
-
-void loop() {
-#ifdef MTP_RX_ENDPOINT
-  mtpd.loop();
-#endif
-  Looper::DoLoop();
-}
+/*

```

Comment:
This diff shows every line in ProffieOS.ino has been changed somehow.
I don't know why that is, but I suspect that maybe line endings have been changed from NL to CRLF-NL
Whatever the reason, it makes it very hard to see what the actual changes are.

Reply:
The only change to the .ino was to include the responsive_styles.h below transitions. I made the changes in Arduino 1.8.9 would that cause?

----added----
// responsive styles
#include "styles/responsive_styles.h"
should I replace with original .ino and make same change again?

Reply:
You need to fix the difff though, or I can't merge the change.